src/fluree/db/connection/config.cljc

```diff
@@ -257,7 +251,7 @@
 (defn keywordize-node-id
   [node]
   (if (subject-node? node)
-    (update node :id iri->kw)
+    (update node "@id" iri->kw)
```

Comment:
Can we use the `const/iri-id` and friends in this file instead of the string literal versions?